src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutorouteContentLocalizationHandler.cs

```diff
@@ -0,0 +1,21 @@
+using System.Threading.Tasks;
+using OrchardCore.Autoroute.Models;
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.ContentLocalization.Handlers;
+
+public class AutorouteContentLocalizationHandler : IContentLocalizationHandler
+{
+    public Task LocalizedAsync(LocalizationContentContext context) => Task.CompletedTask;
+
+    public Task LocalizingAsync(LocalizationContentContext context)
+    {
+        if (context.ContentItem.Has<AutoroutePart>())
+        {
+            // Clearing the AutoroutePart path to regenerate the permalink automatically.
+            context.ContentItem.Content.AutoroutePart.Path = null;
```

Comment:
```suggestion
            context.ContentItem.Alter<AutoroutePart>(p => p.Path = null);
```
I can confirm that this works. Maybe there has been something fixed in the meantime.

Reply:
@gvkries I tried `Alert<TPart>` in another demo I have with @rwxzig a few minutes but it doesn't work, I'm not sure why, could you please confirm again if it's working